conf/machine/hikey-32.conf

```diff
@@ -0,0 +1,45 @@
+#@TYPE: Machine
+#@NAME: LeMaker HiKey machine
+#@DESCRIPTION: LeMaker HiKey machine
+
+require conf/machine/include/tune-cortexa8.inc
+DEFAULTTUNE = "cortexa8hf-neon"
```

Comment:
Did you managed to do a build without setting this? We use a binary Mali driver that needs float compatibility with the rest of the system.  Not setting a default tune to hf-neon also makes the gstreamer recipes choke on linking.